likeminds-chat-reactnative-integration/ChatSX/components/Messages/index.tsx

```diff
@@ -0,0 +1,602 @@
+import { View, Text, Image, TouchableOpacity, Pressable } from "react-native";
+import React, { useEffect, useState, useLayoutEffect, useContext } from "react";
+import { styles } from "./styles";
+import STYLES from "../../constants/Styles";
+import { decode } from "../../commonFuctions";
+import ReplyConversations from "../ReplyConversations";
+import AttachmentConversations from "../AttachmentConversations";
+import ReactionGridModal from "../ReactionGridModal";
+import { useAppDispatch, useAppSelector } from "../../store";
+import {
+  LONG_PRESSED,
+  SELECTED_MESSAGES,
+  SET_POSITION,
+} from "../../store/types/types";
+import { PollConversationView } from "../Poll";
+import { useQuery } from "@realm/react";
+import { myClient } from "../../..";
+import { ChatroomChatRequestState, Events, Keys } from "../../enums";
+import { ChatroomType } from "../../enums";
+import { UserSchemaResponse } from "../../db/models";
+import { USER_SCHEMA_RO } from "../../constants/Strings";
+import LinkPreview from "../LinkPreview";
+import { LMChatAnalytics } from "../../analytics/LMChatAnalytics";
+import { Credentials } from "../../credentials";
+import ReactionList from "../ReactionList";
+import { LMChat } from "../../LMChatProvider";
+
```

Comment:
where is `LMChatBubble`, I believe it has to be a separate UI components which has to be imported here and then used